### DISCUSSION TITLE
Hiding Query Parameters

### DISCUSSION THREAD

**Discussion by mortgagehog** [vercel/next.js#16968] [Help] [answered]
Is there a way to hide query parameters via next link? I would like to create a link on my site that will send you "/?param1=foo&param2=bar" but will show as "/" in the browser? Any help is appreciated. Also, very much appreciate the framework.

**Comment by jamesmosier** [comment, marked as answer]
Hi @mortgagehog. Unfortunately there is no way to do this currently with Next. If you wanted to pass state between pages you might consider using [sessionStorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage)

**Reply by mortgagehog** [reply]
No problem. Thanks for the quick response. Do you know are there any plans to add in this functionality in the future? I am utilizing session storage and didn't want to have to make an extra api requests. BTW Been working with this framework for 5 months now and love it. I don't think I could have built my site so seamlessly and with such good performance without it.

**Reply by jamesmosier** [reply]
I'm not personally aware of that feature coming soon but I also don't know the Next roadmap 😄  glad you've been seeing success using it though 🎉
